pyomo/contrib/mindtpy/tests/unit_test.py

```diff
@@ -10,362 +10,311 @@
 #  ___________________________________________________________________________
 
 """Unit tests for the MindtPy solver."""
-import pyomo.common.unittest as unittest
```

Comment:
Why do you remove these tests? Is it because they rely on the SolveData object? If so and you don't think there's anything that needs rewriting, maybe you could just delete this file.

Reply:
OK. I'll delete this file.